### ISSUE TITLE
Seems to look for Postgresql despite BQ config

### ISSUE THREAD

**Issue by sabman** [Logflare/logflare#2113]
We are trying to run logflare with bigquery and running into the following error when we query the API:

```
https://example.com/analytics/v1/api/endpoints/query/logs.all?project=xxx&iso_timestamp_start=2024-06-23T15:00:00.000Z&sql=select
++cast(timestamp+as+datetime)+as+timestamp,
++event_message,+metadata+
from+edge_logs+
limit+5&ref=xxx&project_tier=ENTERPRISE
```

```
analytics  | 16:41:56.530 [error] GenServer #PID<0.8225.0> terminating
analytics  | ** (RuntimeError) No matching source backend found for Postgres query execution
analytics  |     (logflare 1.7.4) lib/logflare/endpoints.ex:232: anonymous fn/1 in Logflare.Endpoints.exec_query_on_backend/4
analytics  |     (logflare 1.7.4) lib/logflare/endpoints.ex:230: Logflare.Endpoints.exec_query_on_backend/4
analytics  |     (logflare 1.7.4) lib/logflare/endpoints/cache.ex:130: Logflare.Endpoints.Cache.handle_call/3
analytics  |     (stdlib 5.2) gen_server.erl:1131: :gen_server.try_handle_call/4
analytics  |     (stdlib 5.2) gen_server.erl:1160: :gen_server.handle_msg/6
analytics  |     (stdlib 5.2) proc_lib.erl:241: :proc_lib.init_p_do_apply/3
analytics  | Last message (from #PID<0.8223.0>): :query
analytics  |
analytics  | 16:41:56.534 [error] Endpoint query exited for an unknown reason
```

version:
```
image: supabase/logflare:1.7.4
```

config:

```yml
    environment:
      PORT: 4000
      LOGFLARE_NODE_HOST:  127.0.0.1
      DB_USERNAME: ${POSTGRES_USER}
      DB_DATABASE: ${POSTGRES_DB}
      DB_HOSTNAME: ${POSTGRES_HOST}
      DB_PORT: ${POSTGRES_PORT}
      DB_PASSWORD: ${POSTGRES_PASSWORD}
      DB_SCHEMA: _analytics
      LOGFLARE_API_KEY: ${LOGFLARE_API_KEY}
      LOGFLARE_SINGLE_TENANT: true
      LOGFLARE_SUPABASE_MODE: true
      LOGFLARE_LOG_LEVEL: info

      # Comment variables to use Big Query backend for analytics
      # POSTGRES_BACKEND_URL: postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
      # POSTGRES_BACKEND_SCHEMA: _analytics
      # LOGFLARE_FEATURE_FLAG_OVERRIDE: multibackend=true

      # Uncomment to use Big Query backend for analytics
      GOOGLE_PROJECT_ID: ${GOOGLE_PROJECT_ID}
      GOOGLE_PROJECT_NUMBER: ${GOOGLE_PROJECT_NUMBER}
```

**Comment by Ziinc** [comment]
Hi could you do a dump of the `_analytics.backends` table?

**Comment by srasul** [comment]
> Hi could you do a dump of the `_analytics.backends` table?

This table is empty for us:

![image](https://github.com/Logflare/logflare/assets/104100/400c97c5-4bd6-4511-b1e9-100ebd0f19a1)

**Comment by Ziinc** [comment]
What about `_analytics.endpoint_queries` and `_analytics.sources` ? 
There should be records in there.
It is possible that the provisioning did not occur, did you include a http healthcheck against the `/health` path?

**Comment by srasul** [comment]
`_analytics.endpoint_queries` and `_analytics.sources` do have data. I can these these to you if you want.

We do have a healthcheck that checks the `/health` url, and that seems to be ok. `docker ps` says the container sis up and healthy/

**Comment by Ziinc** [comment]
Ah i think i know what you're missing:
```yaml
env:
  LOGFLARE_FEATURE_FLAG_OVERRIDE: multibackend=true
```
this is needed at the moment, as we haven't removed the feature flag.

**Comment by Ziinc** [comment]
Hi i have done a fix for this bug, it will be out in the next release, likely v1.7.15.

**Comment by Ziinc** [comment]
This fix is now in v1.8.0
